Add Organisation, MembersLibrary and update DataStore

inovizz opened the pull request:
- Create Organisation wrt generic DataStore
- Add increaseCount and decreaseCount functions in DataStore
- Add MembersLibrary and its corresponding methods

neeldhwaj commented on contracts/DataStore.sol:
```diff
@@ -10,12 +10,18 @@ import "./helper_contracts/zeppelin/lifecycle/Killable.sol";
-    function addNew() {
+    function increaseCount() {
         // Invoke this function before adding a new record.
         // TODO Find if addNew can be called simultaneously. If yes, the below index will not point to correct entry.
```
Add proper comments

neeldhwaj commented on contracts/DataStore.sol:
```diff
@@ -10,12 +10,18 @@ import "./helper_contracts/zeppelin/lifecycle/Killable.sol";
 
+    function decreaseCount() {
+        // Invoke this function before adding a new record.
+        // TODO Find if addNew can be called simultaneously. If yes, the below index will not point to correct entry.
```
Add proper comments from above

Commit:
Add Organisation, MembersLibrary and update DataStore

- Create Organisation wrt generic DataStore
- Add increaseCount and decreaseCount functions in DataStore
- Add MembersLibrary and its corresponding methods
